### PR TITLE
add preprocess example using experimental dataset 

### PR DESCRIPTION
**Pull request by junchaoxia**
Partially address issue #299.

### CODE REVIEW

File: tutorials/examples/preprocess_imgs_exp [hunk 1]
```diff
@@ -0,0 +1,90 @@
+"""
```
Comment (janden) [inline tutorials/examples/preprocess_imgs_exp]: Change extension to `.py.dontrun`.

File: tutorials/examples/preprocess_imgs_exp [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import logging
+import matplotlib.pyplot as plt
+
+from aspire.source.relion import RelionSource
+from aspire.estimation.noise import WhiteNoiseEstimator
+
+logger = logging.getLogger('aspire')
+
+# Set input path and files and initialize other parameters
+starfile_in = '/tigress/junchaox/CryoEMdata/empiar10028/shiny_2sets.star'
```
Comment (janden) [inline tutorials/examples/preprocess_imgs_exp]: Capital letters for the variable name.

File: tutorials/examples/preprocess_imgs_exp [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+data_folder = '/tigress/junchaox/CryoEMdata/empiar10028/'
+pixel_size = 1.34
+num_imgs = 100
+
+logger.info('This script illustrates how to preprocess experimental CryoEM images')
```
Comment (janden) [inline tutorials/examples/preprocess_imgs_exp]: Just print statements.

File: tutorials/examples/preprocess_imgs_exp [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+logger.info('Obtain original images')
+imgs_od = source.images(start=0, num=nimgs_ext)
+
+logger.info('Perform phase flip to input images')
+source.phase_flip()
+imgs_pf = source.images(start=0, num=nimgs_ext)
+
+max_resolution = 60
+logger.info(f'Downsample resolution to {max_resolution} X {max_resolution}')
+if max_resolution < source.L:
```
Comment (janden) [inline tutorials/examples/preprocess_imgs_exp]: No need for if statement.

**Review by janden — CHANGES_REQUESTED** [review]

File: tutorials/examples/preprocess_imgs_exp (hunk 1 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp]: Done.

File: tutorials/examples/preprocess_imgs_exp (hunk 2 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp]: Done.

File: tutorials/examples/preprocess_imgs_exp (hunk 4 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp]: Done.

File: tutorials/examples/preprocess_imgs_exp (hunk 3 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp]: Done.

File: tutorials/examples/preprocess_imgs_exp.py.dontrun [hunk 5]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env  python3.6
```
Comment (vineetbansal) [inline tutorials/examples/preprocess_imgs_exp.py.dontrun]: #!/usr/bin/env  python

File: tutorials/examples/preprocess_imgs_exp.py.dontrun [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+"""
+
+import matplotlib.pyplot as plt
+
+from aspire.source.relion import RelionSource
+from aspire.estimation.noise import WhiteNoiseEstimator
+
+
+# Set input path and files and initialize other parameters
+data_folder = '/path/to/untarred/empiar/dataset/'
```
Comment (vineetbansal) [inline tutorials/examples/preprocess_imgs_exp.py.dontrun]: I guess make all of these capitalized as well.

File: tutorials/examples/preprocess_imgs_exp.py.dontrun (hunk 5 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp.py.dontrun]: Done.

File: tutorials/examples/preprocess_imgs_exp.py.dontrun (hunk 6 above)
Comment (junchaoxia) [reply tutorials/examples/preprocess_imgs_exp.py.dontrun]: Done in new commit f0db5cd.

**Review by janden — APPROVED** [review]

**Pull request merged**